examples/flows/chat/basic-chat/flow.dag.yaml

```diff
@@ -15,6 +15,7 @@ outputs:
 nodes:
 - inputs:
     deployment_name: gpt-35-turbo
+    model: gpt-3.5-turbo
```

Comment:
Shall we add a comment this is for easily switch between openai and azureopenai

Reply:
add comment